docs/docusaurus/versioned_docs/version-1.5.X/nms/architecture_overview.md

```diff
@@ -52,7 +52,7 @@ Session middleware helps with creating the session, setting the session cookie(u
 
 **csrfMiddleware**
 
-This protects against any CSRF attacks.  For more understanding, check this (https://github.com/pillarjs/understanding-csrf#csrf-tokens). Magmalte server includes the csrf tokens when it responds to the client and client submits the form with the token. (nms/app/packages/magmalte/app/common/axiosConfig.js)
+This protects against any CSRF attacks.  For more understanding, check this (https://github.com/pillarjs/understanding-csrf#csrf-tokens). Magmalte server includes the csrf tokens when it responds to the client and client submits the form with the token. (nms/app/packages/magmalte/app/axiosConfig.js)
```

Comment:
Version 1.5 docs should refer to the v1.5 branch of the codebase, so this change shouldn't be included in this versioned doc.